src/java/picard/analysis/GcBiasMetricsCollector.java

```diff
@@ -191,7 +191,7 @@ public void addMetricsToFile(final MetricsFile<GcBiasMetrics, Integer> file) {
                 final long[] errorsByGc = gcCur.errorsByGc;
                 final long[] basesByGc = gcCur.basesByGc;
                 final int totalClusters = gcCur.totalClusters;
-                final int totalAlignedReads = gcCur.totalAlignedReads;
+                final long totalAlignedReads = gcCur.totalAlignedReads;
```

Comment:
This may not be the best decision, but the reason I was able to figure out the bug was because the old version (aka pre-single pass sam) of gc bias didn't have this check, so it calculated all gc bias metrics as normal and the only anomaly in the output was the overflowed totalAlignedReads value. I thought it'd be better to have the gc bias program functioning as closely to the old version as possible.

I tried running it without the check altogether but when there are no aligned reads to a specific accumulation level the 'group' ends up being null and it threw errors from that. 

I decided on the skip only if it equaled zero as a compromise, so that when there's really no aligned reads it doesn't have an issue but it would still function similarly to the old version. 

What do you think is best?